intranet/apps/auth/views.py

```diff
@@ -200,11 +200,14 @@ def post(self, request):
                     today_8 = Day.objects.today().day_type.blocks.filter(name__contains="8")
                     if today_8:
                         first_start_time = time(today_8[0].start.hour, today_8[0].start.minute)
+                        last_block = today_8.pop()
```

Comment:
'QuerySet' object has no attribute 'pop'

```suggestion
                        last_block = today_8.last()
```

Perhaps you may want to revert and go back to `today_8.last().start.hour` and the same for the minute?

Reply:
Trying to figure out why my dev env didn't catch this..ugh

Reply:
I have a feeling it is because of the try-catch used; I had to do a `print` to get the exception that was thrown